<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceWindow</class>
 <widget class="QMainWindow" name="DeviceWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manager devices</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="deviceNameText">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Choice device</string>
    </property>
   </widget>
   <widget class="QLabel" name="deviceTypeText">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Choice device</string>
    </property>
   </widget>
   <widget class="QTreeView" name="datasetTreeView">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>261</width>
      <height>401</height>
     </rect>
    </property>
    <property name="autoExpandDelay">
     <number>1</number>
    </property>
    <property name="indentation">
     <number>10</number>
    </property>
   </widget>
   <widget class="QTreeView" name="libraryTreeView">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>261</width>
      <height>161</height>
     </rect>
    </property>
    <property name="indentation">
     <number>4</number>
    </property>
    <property name="rootIsDecorated">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTreeView" name="analysisTreeView">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>220</y>
      <width>261</width>
      <height>191</height>
     </rect>
    </property>
    <property name="autoExpandDelay">
     <number>-1</number>
    </property>
    <property name="indentation">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Type:</string>
    </property>
   </widget>
   <widget class="QLabel" name="deviceImage">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>180</width>
      <height>227</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="addLibraryButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>180</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addAnalysisButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>420</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addMeasureButton">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>420</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Model:</string>
    </property>
   </widget>
   <widget class="QLabel" name="deviceModelText">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Choice device</string>
    </property>
   </widget>
   <widget class="QPushButton" name="runExtractButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>410</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Run Extraction</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>810</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDevice">
    <property name="title">
     <string>Devices</string>
    </property>
    <addaction name="actionDeviceOpen"/>
    <addaction name="actionDeviceRemove"/>
    <addaction name="actionDeviceClose"/>
    <addaction name="actionBuildGraphics"/>
   </widget>
   <widget class="QMenu" name="menuMeasures">
    <property name="title">
     <string>Measures</string>
    </property>
    <addaction name="actionAddMeasure"/>
    <addaction name="actionEditMeasure"/>
   </widget>
   <widget class="QMenu" name="menuParameters">
    <property name="title">
     <string>Parameters</string>
    </property>
    <addaction name="actionEditorLibrary"/>
   </widget>
   <widget class="QMenu" name="menuExtraction">
    <property name="title">
     <string>Extraction</string>
    </property>
    <addaction name="actionExtractionRun"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionSettingsOpen"/>
    <addaction name="actionAboutOpen"/>
   </widget>
   <addaction name="menuDevice"/>
   <addaction name="menuMeasures"/>
   <addaction name="menuParameters"/>
   <addaction name="menuExtraction"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">float:right;</string>
   </property>
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <action name="actionDeviceOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionDeviceRemove">
   <property name="text">
    <string>Remove</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionDeviceClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionEditorLibrary">
   <property name="text">
    <string>Editor</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionExtractionRun">
   <property name="text">
    <string>Run</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionAddMeasure">
   <property name="text">
    <string>Add</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+M</string>
   </property>
  </action>
  <action name="actionEditMeasure">
   <property name="text">
    <string>Edit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionSettingsOpen">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionAboutOpen">
   <property name="text">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionBuildGraphics">
   <property name="text">
    <string>Graphics</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
